src/command.c

```diff
@@ -15186,10 +15185,62 @@ char WhichStand (int x)
         return ('8');
     else if (x == 512)
         return ('9');
-    else if (x > 0 && x < 512)
+    else if (x == 1001)
+        return ('a');
+    else if (x == 1002)
+        return ('b');
+    else if (x == 1003)
+        return ('c');
+    else if (x == 1004)
+        return ('d');
+    else if (x == 1005)
+        return ('e');
+    else if (x == 1006)
+        return ('f');
+    else if (x == 1007)
+        return ('g');
+    else if (x == 1008)
+        return ('h');
+    else if (x == 1009)
+        return ('i');
+    else if (x == 1010)
+        return ('j');
+    else if (x == 1011)
+        return ('k');
+    else if (x == 1012)
+        return ('l');
+    else if (x == 1013)
+        return ('m');
+    else if (x == 1014)
+        return ('n');
+    else if (x == 1015)
+        return ('o');
+    else if (x == 1016)
+        return ('p');
+    else if (x == 1017)
+        return ('q');
+    else if (x == 1018)
+        return ('r');
+    else if (x == 1019)
+        return ('s');
+    else if (x == 1020)
+        return ('t');
+    else if (x == 1021)
+        return ('u');
+    else if (x == 1022)
+        return ('v');
+    else if (x == 1023)
+        return ('w');
+    else if (x == 1024)
+        return ('x');
+    else if (x == 1025)
+        return ('y');
+    else if (x == 1026)
+        return ('z');
+    else if (x > 0 && x < 1000000)
```

Comment:
Why are we not using bits to represent all states? This appears likely to cause errors elsewhere in the code. For instance, will the parsimony-guided moves now compute the correct parsimony lengths with this coding? I doubt it.

Reply:
again this is old code, have been using bits in the latest code